hackclub.com.yaml

```diff
@@ -1,4327 +1,4485 @@
 ham:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 hardware:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 harvest:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 haxidraw:
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
-
 hcarc:
-  - type: MX
-    ttl: 600
-    values:
-      - exchange: ASPMX.L.GOOGLE.COM.
-        preference: 1
-      - exchange: ALT1.ASPMX.L.GOOGLE.COM.
-        preference: 5
-      - exchange: ALT2.ASPMX.L.GOOGLE.COM.
-        preference: 5
-      - exchange: ALT3.ASPMX.L.GOOGLE.COM.
-        preference: 10
-      - exchange: ALT4.ASPMX.L.GOOGLE.COM.
-        preference: 10
-  - type: A
-    ttl: 600
-    value: 76.76.21.21
-  - type: TXT
-    ttl: 600
-    values:
-      - google-site-verification=xzGbA1OB1AnLPBHMOjrh5qPFZIwXXe-1_ci2GuomAjY
-      - v=spf1 include:_spf.google.com -all
-
+- ttl: 300
+  type: MX
+  values:
+  - exchange: ASPMX.L.GOOGLE.COM.
+    preference: 1
+  - exchange: ALT1.ASPMX.L.GOOGLE.COM.
+    preference: 5
+  - exchange: ALT2.ASPMX.L.GOOGLE.COM.
+    preference: 5
+  - exchange: ALT3.ASPMX.L.GOOGLE.COM.
+    preference: 10
+  - exchange: ALT4.ASPMX.L.GOOGLE.COM.
+    preference: 10
+- ttl: 300
+  type: A
+  value: 76.76.21.21
+- ttl: 300
+  type: TXT
+  values:
+  - google-site-verification=xzGbA1OB1AnLPBHMOjrh5qPFZIwXXe-1_ci2GuomAjY
+  - v=spf1 include:_spf.google.com -all
 hcer-public-github-data:
-  - type: CNAME
-    value: a.selfhosted.hackclub.com.
-
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 heist:
-  - ttl: 600
-    type: CNAME
-    value: heist.iunstable0.com.
-
+- ttl: 300
+  type: CNAME
+  value: heist.iunstable0.com.
 heist-api:
-  - ttl: 600
-    type: CNAME
-    value: heist-api.iunstable0.com.
-
+- ttl: 300
+  type: CNAME
+  value: heist-api.iunstable0.com.
 _dmarc.hcarc:
+  ttl: 300
   type: TXT
-  ttl: 600
-  value: "v=DMARC1\\; p=quarantine\\;"
+  value: v=DMARC1\; p=quarantine\;
 _dmarc.scrapyard:
+  ttl: 300
   type: TXT
-  ttl: 3600
   value: v=DMARC1\; p=none\; rua=mailto:dmarc_report@hackclub.com\; ruf=mailto:dmarc_report@hackclub.com\;fo=1
 key1._domainkey.hcarc:
+  ttl: 300
   type: CNAME
-  ttl: 600
   value: key1.hcarc.hackclub.com._domainkey.migadu.com.
 key2._domainkey.hcarc:
+  ttl: 300
   type: CNAME
-  ttl: 600
   value: key2.hcarc.hackclub.com._domainkey.migadu.com.
 key3._domainkey.hcarc:
+  ttl: 300
   type: CNAME
-  ttl: 600
   value: key3.hcarc.hackclub.com._domainkey.migadu.com.
-      
 hcb:
-  - type: ALIAS
-    value: ingress.hcb.hackclub.com.
-  - type: MX
-    ttl: 6000
-    value:
-      exchange: mx.sendgrid.net.
-      preference: 10
-  - ttl: 3600
-    type: TXT
-    values:
-      - google-gws-recovery-domain-verification=49632973
-      - yandex-verification=befd33a8f4f21226
```

Comment:
This MX record should not have been deleted! This is a breaking change, was this communicated to the team responsible?